### DISCUSSION TITLE
use custom domain for images link

### DISCUSSION THREAD

**Discussion by phankhanhpvk** [outline/outline#5827] [General]
How i can use custom link for image storage in s3?
like if acl public-read image upload then use custom domain for images (for cloudflare cache)

**Comment by tommoor** [comment]
You can achieve this with `AWS_S3_ACCELERATE_URL` env config, it basically acts as a CDN url for all downloads

**Reply by phankhanhpvk** [reply]
that means i need to use s3 accelerate service?

**Reply by tommoor** [reply]
No, it's just a badly named variable.

**Reply by phankhanhpvk** [reply]
when i add domain use with cloudflare proxy for `AWS_S3_ACCELERATE_URL` i have error
`<Code>AccessDenied</Code><Message>Invalid according to Policy: Policy Condition failed: ["eq", "$bucket", "outline"]</Message>`
